git-open.1.md

```diff
@@ -0,0 +1,138 @@
+# git-open - Open the repository's website in your browser.
+
+
+## SYNOPSIS
+
+`git-open` [remote-name] [branch-name]
+
+`git-open` --issue
+
+
+## DESCRIPTION
+
+`git-open` opens the repository's website in your browser. The major well known
+git hosting services are supported.
+
+
+## OPTIONS
+
+
+`-i`, `--issue`
+  Open the current issue. When the name of the current branch matches the right pattern, 
+  it will open the webpage with that issue. See `EXAMPLES` for more information. 
+  This only works on GitHub, GitLab, Visual Studio Team Services and Team Foundation Server at the moment.
+
+`-h`
+  Show a short help text.
+
+
+## EXAMPLES
+
+```sh
+$ git open
+```
+
+It opens https://github.com/TRACKED_REMOTE_USER/CURRENT_REPO/
+
+```sh
+$ git open someremote
+```
+
+It opens https://github.com/PROVIDED_REMOTE_USER/CURRENT_REPO/
+
+```sh
+$ git open someremote somebranch
+```
+
+It opens https://github.com/PROVIDED_REMOTE_USER/CURRENT_REPO/tree/PROVIDED_BRANCH
+
+```sh
+$ git open --issue
+```
+
+If branches use naming convention of `issues/#123`, it opens
+https://github.com/TRACKED_REMOTE_USER/CURRENT_REPO/issues/123
+
+
+## SUPPORTED REMOTE REPOSITORIES
+
+git-open can automatically guess the corresponding repository page for remotes
+(default looks for `origin`) on the following hosts:
+
+- github.com
+- gist.github.com
+- gitlab.com
+- GitLab CE/EE (self hosted GitLab, see `CONFIGURATION`)
+- bitbucket.org
+- Atlassian Bitbucket Server (formerly _Atlassian Stash_)
+- Visual Studio Team Services
+- Team Foundation Server (on-premises)
+
+
+## CONFIGURATION
+
+To configure git-open you may need to set some `git config` options. 
+You can use `--global` to set across all repos, instead of just the local one.
+
+```sh
+$ git config [--global] option value
+```
+
+### Configuring which remote to open 
+
+By default, `git open` opens the remote named `origin`. However, if your current branch is remotely-tracking a different remote, that tracked remote will be used.
```

Comment:
I see now.. maybe instead say "`git open` will fall back to use `origin` if your current branch is not remotely-tracking a different remote."